asmcli/lib/installation_dependencies.sh

```diff
@@ -185,6 +191,10 @@ enable_gcloud_apis(){
   info "Enabling required APIs..."
   # shellcheck disable=SC2046
   retry 3 gcloud services enable --project="${PROJECT_ID}" $(required_apis | tr '\n' ' ')
+  if is_managed; then
+    # shellcheck disable=SC2046
+    retry 3 gcloud services enable --project="${PROJECT_ID}" $(old_managed_required_apis | tr '\n' ' ')
```

Comment:
I thought we didn't need to explicitly enable this, since it's auto-enabled by `mesh`.

Reply:
Thanks for the catch! Removed.